app/constants/welcome.js

```diff
@@ -2,51 +2,61 @@ export const welcome = [
   {
     lang: 'English',
     welcome: 'Welcome',
-    nextBtn: 'Next'
+    nextBtn: 'Next',
+    selectLan: 'Select your language'
```

Comment:
Let it be selectLang instead of selectLan
```suggestion
    selectLang: 'Select your language'
```

Reply:
OK i'll do it